### ISSUE TITLE
update to phantomjs 2

### ISSUE THREAD

**Issue by naugtur** [ForbesLindesay/run-browser#15]
It's high time we stopped dealing with the oldest webkit alive.

**Comment by cain-uber** [comment]
Yes please! Phantom 2.0 has many bug fixes that can make our tests easier.

**Comment by malandrew** [comment]
I want this too. Almost there:
https://github.com/ariya/phantomjs/issues/12948
